centos.dockerfile

```diff
@@ -5,12 +5,21 @@ ARG BIN_IMAGE=denoland/deno:bin-${DENO_VERSION}
 FROM ${BIN_IMAGE} AS bin
 
 
+FROM buildpack-deps:20.04-curl AS tini
+
+ARG TINI_VERSION=0.19.0
+RUN curl -fsSL https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini \
```

Comment:
@hayd this would be the same as asking to add the tini binary to the deno.zip in the GitHub Releases. The `denoland/debo:bin` is supposed to contain deno's binary as the name implies, not tini's.

If all you want is to prevent having to repeat the code which downloads tini, I'm working on a PR for it as well (as I mentioned before).